libraries/websphere_base.rb

```diff
@@ -193,6 +193,50 @@ def enable_as_service(service_name, srvr_name, prof_path, runas = '')
               runas_user: runas
             )
           end
+
+          node_srvc = srvr_name == 'dmgr' ? 'Manager' : 'Server'
```

Comment:
does the template need to call `systemctl daemon-reload`?

Reply:
yeah it probably should, updated